sycl/include/sycl/ext/oneapi/matrix/matrix-unified.hpp

```diff
@@ -63,15 +63,15 @@ class wi_data {
 #if defined(__NVPTX__)
     return jm.cuda_impl.wi_marray.size();
 #else
-    return __spirv_JointMatrixWorkItemLengthINTEL(jm.spvm);
+    throw runtime_error("Use the correct namespace.", PI_ERROR_INVALID_DEVICE);
 #endif
   };
 
   decltype(auto) operator[](size_t i) {
 #if defined(__NVPTX__)
     return (jm.cuda_impl.wi_marray[i]);
 #else
-    return wi_element<T, Rows, Cols, Use, Layout, Group>(jm, i);
+    throw runtime_error("Use the correct namespace.", PI_ERROR_INVALID_DEVICE);
```

Comment:
a better message would be:
"get_wi_data is available using:  ext::intel::experimental::matrix::get_wi_data"

Reply:
Done